src/interface_adapter/login/LoginController.java

```diff
@@ -2,18 +2,27 @@
 
 import use_case.login.LoginInputBoundary;
 import use_case.login.LoginInputData;
+import use_case.switch_view.SwitchViewInputBoundary;
+import use_case.switch_view.SwitchViewInputData;
 
 public class LoginController {
 
   final LoginInputBoundary loginUseCaseInteractor;
+  private final SwitchViewInputBoundary switchViewInteractor;
 
-  public LoginController(LoginInputBoundary loginUseCaseInteractor) {
+  public LoginController(
+      LoginInputBoundary loginUseCaseInteractor, SwitchViewInputBoundary switchViewInteractor) {
     this.loginUseCaseInteractor = loginUseCaseInteractor;
+    this.switchViewInteractor = switchViewInteractor;
```

Comment:
Aren't we expected to have a SwitchViewController?

Reply:
Not for a use case